CamBench_Cap/src/main/java/org/cambench/cap/mixedsensitivities/inter2path/falsepositive/brokencrypto/BrokenCrypto1_meta1.yaml

```diff
@@ -5,7 +5,7 @@ api:
   javax.crypto.Cipher
 
 description:
-  A string containing the name of an secure cipher (AES) is passed to a in the reachable branch of a 
+  A string containing the name of a secure cipher (AES) is passed to a in the reachable branch of a 
```

Comment:
Correction of the sentence (article duplication): 
... is passed to the reachable branch of a conditional statement.